### ISSUE TITLE
unregcognized line [tunnelled-dir-server]

### ISSUE THREAD

**Issue by iwakeh** [tomlurge/converTor#10]
```
 Unrecognized lines in bridge-descriptors/server-descriptors/2016-06-10-12-14-06-server-descriptors:
 [tunnelled-dir-server]
```

When converting bridge descriptors.

```
@type bridge-server-descriptor 1.2
router orak2 10.113.214.210 53145 0 0
or-address [fd9f:2e19:3bcf::89:2281]:53145
master-key-ed25519 z0J8Zq5PcWXvc3jVo0WMeq8scYgrxEgZJ1AfNbxR+XU
platform Tor 0.2.8.0-alpha-dev on Linux
protocols Link 1 2 Circuit 1
published 2015-12-31 19:30:53
fingerprint D0A5 101B FAA5 4488 6E3E AA14 7A53 F7C0 BC44 3015
uptime 611885
bandwidth 14971520 104857600 60761
extra-info-digest B080679F850A66E158A2980C21DBA22A5E1DA0FATQK+knF6YRJ5/vZn1Bzsa8dPTKB/bJYN/r66h22s4QM
hidden-service-dir
contact somebody
ntor-onion-key v9mbXskVFrZWzW7NmNSEwZXsjxQBpO3TMhPmXzZ3gB4=
reject *:*
tunnelled-dir-server
router-digest-sha256 +XcMyHZ0uVmD+bl3N/2KlTbitmk0YWoIGMlHtrKVB0c
router-digest C66128086F0636C47FD0691DC7888A73DADBA7E6
```

**Comment by tomlurge** [comment]
this is already reported as an issue of metrics-lib:
https://trac.torproject.org/projects/tor/ticket/19284

**Comment by iwakeh** [comment]
Will be fixed with upcoming release 1.3.0 of metrics-lib :-)
Already available as pre-release: https://trac.torproject.org/projects/tor/ticket/19398
